doc/whats-new.rst

```diff
@@ -42,6 +42,9 @@ Bug fixes
   ``keep_attrs=True`` option. By
   `Jeremy McGibbon <https://github.com/mcgibbon>`_.
 
+- ``decode_cf_timedelta`` now accepts arrays with ``ndim`` >1 (pydata/xarray/pull/842).
```

Comment:
don't forget to add your name in the credits!

also, reference the issue like this:

    :issue:`842`